src/test/java/com/razorpay/IinClientTest.java

```diff
@@ -54,4 +56,27 @@ public void fetch() throws RazorpayException {
             assertTrue(false);
         }
     }
+
+    @Test
+    public void fetchList() throws RazorpayException {
+
+        JSONObject request = new JSONObject();
+        request.put("flow","otp");
+
+        JSONObject mockedResponseJson = new JSONObject();
+        mockedResponseJson.put("count", 0);
+        mockedResponseJson.put("entity", "iin");
+        JSONArray iinArr = new JSONArray();
```

Comment:
make it iinArray instead for better readability

Reply:
made typo correction

Reply:
https://github.com/razorpay/razorpay-java/blob/020d762e5d69323acbd579dabe42a79e2844de4f/src/test/java/com/razorpay/IinClientTest.java#L69

Reply:
+1